README.md

```diff
@@ -1,5 +1,11 @@
 ## MongoDB Atlas CLI
 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/5663078/161264663-6c059196-6c20-4efc-980a-6266f7017917.png" alt="CLI Logo">
```

Comment:
can you try this 
```suggestion
<picture>
    <source srcset="atlas_cli_inverse.png" media="(prefers-color-scheme: dark)">
    <img src="atlas_cli.png">
</picture>
```
not sure if git hub will support such a fancy thing

Reply:
(my url are wrong but I hope you get the idea, img with the normal image, prefers dark for the alternative inverse version of the icon

Reply:
I have just updated the pr with your proposal but it doesn’t seem to work.

Reply:
it should be possible thought  https://stackoverflow.com/questions/65413712/changing-readme-md-image-display-conditional-to-github-light-mode-dark-mode